content/en/docs/guide_openapi/upgrading.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Upgrading from endpoints
+weight: 90
+date: 2022-03-01
+description: If you are an existing customer, you may be familiar with [Swagger endpoints](/docs/developer_guide/flows/manage_endpoints) and have existing applications that you wish to upgrade to use **OpenAPI** flow-trigger. This document describes how to upgrade.
+---
+
+## Overview
+
+While significant effort was made to make the new **OpenAPI** flow-trigger as compatible with [Swagger endpoints](/docs/developer_guide/flows/manage_endpoints) interface as possible, there are a number of key differences that were made for better design and UX, that may affect your application, or clients that use it.
+
+One of the main driving forces for this change is that the modules we used to support Swagger endpoints are no longer maintained and they only support OpenAPI 2.0. That, and combined with architecture changes, it is now possible to support pluggable modules that can support HTTP and API-first, such as OpenAPI. **OpenAPI** flow-trigger provides a better experience and support.
+
+The following document should help you upgrade your application and guide you through any issues you may encounter.
+
+{{% alert title="Deprecation" color="warning" %}}
+Support for [Swagger endpoints](/docs/developer_guide/flows/manage_endpoints) will end on **31 March 2023**.
+{{% /alert %}}
+
+## Upgrading to OpenAPI flow-trigger
+
+Before upgrading, we recommend the following:
+
+1. Read this guide and get familiar with the [known differences](#known-differences) between Swagger endpoints and the new **OpenAPI**. For most applications, this upgrade should be very simple.
+1. Ensure that your application currently works as expected, and that all code paths are covered by tests. It will be much easier to test any change, including upgrades, if adequate tests are in place.
+1. Back up your application.
+
+Open a command prompt and change to the directory of the {{% variables/apibuilder_prod_name %}} project that you wish to upgrade. Note that this command will upgrade your application, and re-install your `node_modules` using npm.  If you do not want this, then you can provide the `--no-install` option:
+
+```bash
+npx @axway/api-builder-openapi-upgrade
+```
+
+After running, the command, you should see output similar to this:
+
+```text
+API Builder OpenAPI upgrade, version 1.0.0
+
+✔ Created: /home/jamie/piper/rdpp-swagger/apidocs/openapi/myapi.json
+✔ Skipped installation, please run "npm install" manually
```

Comment:
It won't say skipped installation when running npx @axway/api-builder-openapi-upgrade